ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -722,6 +722,21 @@ static void setPreVote(RaftProperties properties, boolean enablePreVote) {
     }
   }
 
+  interface ReadOnly {
```

Comment:
Let's call it "Read" instead of "ReadOnly".  Please move it to right before the Write interface, i.e. line 159.

Reply:
done